apps/api-auth/src/modules/auth/auth.controller.ts

```diff
@@ -11,7 +11,7 @@ import {
     VerifyEmailDto,
 } from './dto';
 
-@Controller('auth')
+@Controller('/')
```

Comment:
Add @ApiTags to every controller, and @ApiOperation with a summary to each controller method. Ideally, also add @ApiResponse for successful responses. Response classes (with @ApiProperty fields) can be created in the ./responses folder.
```suggestion
@ApiTags('Auth')
@Controller('/')
...
@ApiOperation({ summary: 'Change password' })
@ApiResponse({
		status: 200,
		type: ChangePasswordResponse,
	})
@Post('change-password')
    changePassword(@Body() body: ChangePasswordDto) {
        return this.authService.changePassword(body.userId, body.oldPassword, body.newPassword);
    }
```

Reply:
Done